src/main/java/uk/gov/hmcts/reform/feature/config/SecurityConfiguration.java

```diff
@@ -68,8 +68,8 @@ protected void configure(HttpSecurity http) throws Exception {
             http
                 .antMatcher("/api/ff4j/**")
                 .authorizeRequests()
-                .antMatchers(HttpMethod.GET).hasRole("USER")
-                .antMatchers(HttpMethod.OPTIONS).hasRole("USER")
+                .antMatchers(HttpMethod.GET).permitAll()
+                .antMatchers(HttpMethod.OPTIONS).permitAll()
                 .antMatchers(HttpMethod.DELETE).hasRole("EDITOR")
```

Comment:
may be we should put EDITOR in constant.Not part of PR though

Reply:
Once the updated user access is in place (the work you are doing) it'll be clearer what to put where :wink: